src/index.ts

```diff
@@ -28,6 +28,7 @@ export class ParifiSdk {
     subgraphConfig: SubgraphConfig,
     relayerConfig: RelayerConfig,
     pythConfig: PythConfig,
+    rewardList: RewardList,
```

Comment:
I'm still not sure if this is the proper approach to do this, as the sdk will need to be reinitialized every time we need the updated reward list. Ideally this should be a param to the functions.
However, as we need to roll this out asap, we can continue with this approach for now, and revisit this later.

 This should be an optional config.